infra/pySourceCredChart/js/lib/example.js

```diff
@@ -0,0 +1,49 @@
+var widgets = require('@jupyter-widgets/base');
```

Comment:
This looks like a leftover file from a hello-world example? remove?

Reply:
removed